### PR TITLE
Metrics for Stats.CollectAzureCdnLogs job

### PR DESCRIPTION
**Pull request by agr**
Related to https://github.com/NuGet/Engineering/issues/4596

For each blob it consumes it now reports its size, number of lines and time it took to process.

Consider turning off showing whitespace difference when viewing, it'll make the diff simpler.

### CODE REVIEW

File: src/Stats.CollectAzureCdnLogs/Job.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.Design;
+using System.Diagnostics;
```
Comment (agr) [inline src/Stats.CollectAzureCdnLogs/Job.cs:7]: Need to remove.

**Comment by joelverhagen** [comment]
Is this meant to be a hotfix? Seems okay if so, just wanted to confirm.

File: src/Stats.CollectAzureCdnLogs/TelemetryService.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ namespace Stats.CollectAzureCdnLogs
     public class TelemetryService : ITelemetryService
     {
         private const string Prefix = "Stats.CollectAzureCdnLogs.";
-
+        private const string FilenamePropertyName = "Filename";
```
Comment (joelverhagen) [inline src/Stats.CollectAzureCdnLogs/TelemetryService.cs:17]: super nit: choose a consistent usage of fileName vs. filename

**Review by joelverhagen — APPROVED** [review]

**Review by lyndaidaii — APPROVED** [review]

**Review by joelverhagen — APPROVED** [review]

**Comment by mariaghiondea** [comment]
@agr is this still needed? Ready to merge?

**Comment by agr** [comment]
@mariaghiondea that should be useful, but I'll need to test it again.

**Pull request closed without merging**